src/sharedServices/dynamoDB/dynamoDBUpdate.ts

```diff
@@ -42,8 +41,4 @@ export const updateQueryByZendeskId = async (
       `Failed to update item in db for zendesk ticket: ${zendeskId}`
     )
   }
-
-  logger.info(`Updated item in db: ${JSON.stringify(responseObject)}`)
```

Comment:
Probably still useful to know a DB update was successful? Don't need the response object though.

Reply:
We removed this on the basis that a failed update would be logged so it may not be necessary to log a successful update

Reply:
Dealt with now by the fact we log out after every successful SDK operation